README.md

```diff
@@ -1,8 +1,8 @@
 # Metabase
 
-[Metabase](https://www.metabase.com) is the easy, open-source way for everyone in your company to ask questions and learn from data.
+[Metabase](https://www.metabase.com) is the easy, open-source way for your company to ask questions and learn from data.
```

Comment:
```suggestion
[Metabase](https://www.metabase.com) is the easy, open-source way for everyone in your company to ask questions and learn from data.
```
Why drop "everyone"? That's Metabase's big deal; anyone can actually get answers from their data.